libs/partners/cohere/langchain_cohere/cohere_agent.py

```diff
@@ -83,8 +88,12 @@ def _convert_to_cohere_tool(
             description=tool.description,
             parameter_definitions={
                 param_name: ToolParameterDefinitionsValue(
-                    description=param_definition.get("description"),
-                    type=param_definition.get("type"),
+                    description=param_definition.get("description")
+                    if "description" in param_definition
+                    else "",
+                    type=JSON_TO_PYTHON_TYPES.get(
```

Comment:
reuse `get_type` from `libs/partners/cohere/langchain_cohere/react_multi_hop/agent.py` for consistency